03_launch_mgmt_cluster.sh

```diff
@@ -107,6 +107,10 @@ IRONIC_ENDPOINT=${IRONIC_URL}
 IRONIC_INSPECTOR_ENDPOINT=${IRONIC_INSPECTOR_URL}
 EOF
 
+  if [ "$NODES_PLATFORM" == "libvirt" ] ; then
+    echo "IRONIC_KERNEL_PARAMS=console=ttyS0" | sudo tee -a "${BMOPATH}/config/default/ironic.env"
+  fi
+
```

Comment:
In this case we are not deploying ironic, only BMO, so this is not needed here (See the "true false" on the following line)

Reply:
thanks, fixed